plugin/trino-hive/src/test/java/io/trino/plugin/hive/optimizer/TestHivePlans.java

```diff
@@ -165,7 +164,7 @@ public void testPrunePartitionLikeFilter()
                                                 project(
                                                         filter("\"like\"(L_STR_PART, \"$like_pattern\"('t%'))",
                                                                 tableScan("table_str_partitioned", Map.of("L_INT_COL", "int_col", "L_STR_PART", "str_part"))))),
-                                        exchange(LOCAL, REPARTITION,
```

Comment:
now that you kept new matcher, it makes sense to use 
```
exchange(LOCAL,
  ...
```
e.g it's either you keep the matcher and use it here, or remove matcher and just match exchange

Reply:
ok, lets match LOCAL here